packages/website/docs/creating-multi-source-autocompletes.md

```diff
@@ -1,4 +1,361 @@
 ---
-id: creating-multi-source-autocompletes
-title: Creating multi-source autocompletes
+id: adding-multiple-categories
+title: Adding multiple categories
```

Comment:
I'd rename this "Adding multiple sources". We use the word "categories" for other meanings (QS categories, etc.).

Reply:
I agree category might not be the best, but am also not sure we should use "sources." In trying to be customer/problem focused, there first thought might not be "how do I add multiple sources?" since this is an autocomplete term.

What about "adding multiple result types?"

Reply:
Yes, that's better.